### DISCUSSION TITLE
Evetn vises ikke på portal forside

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#13] [Bugs] [answered]
Dette event vises ikke på portalforside:

![image](https://github.com/user-attachments/assets/0d182704-a6c4-4006-acf8-46b310a0ac68)

Hvis jeg vælger portalforsiden, som viser aktuelle evnets dags dato, så dukker den ikke op:
![image](https://github.com/user-attachments/assets/e914c5d9-ab91-4dc5-9ba9-7ef7b19d21d7)

Hvis jeg vælger dato fra 8. oktober (som er i dag), så dukker den op:

![image](https://github.com/user-attachments/assets/06960d27-5af4-4391-a12a-1b7afbc52878)

Ligeledes vælger jeg arrangørsiden, så fremgår den: 

![image](https://github.com/user-attachments/assets/4845984d-62ac-4ecd-b7fa-fb6590d5ffc3)

**Comment by henrikbak** [comment, marked as answer]
Den fremgår af forsiden nu, så jeg vil gætte på at det er fordi visningen af forsiden har været cachet. Det kan nogle gange kræve et hard refresh af siden, før man ser de seneste ændringer.
